tests/test_execution_subprocess_executor.py

```
@@ -0,0 +1,206 @@
+from __future__ import annotations
+
+import asyncio
+import os
+import sys
+import time
+from pathlib import Path
+
+import pytest
+
+
+def _ctx(tmp_path: Path) -> "JobContext":
```

Ruff (0.15.10)

[error] Undefined name `JobContext`

(F821)